### PR TITLE
#fixes remove settings and schedule in  nats publish

### PR DESCRIPTION
**Pull request by Linoy339**

### CODE REVIEW

File: src/service/ActivityService.ts [hunk 1]
```diff
@@ -11,9 +11,9 @@ export class ActivityService {
   public static _name = "Activity"
   public static Router = Router()
 
-  public static async list(auth: any, study_id: string, ignore_binary: boolean, sibling: boolean = false) {
+  public static async list(auth: any, study_id: string, ignore_binary: boolean, sibling = false) {
```
Comment (avaidyam) [inline src/service/ActivityService.ts:14]: Please **DO NOT** remove typing information.

Comment (Linoy339) [reply src/service/ActivityService.ts:14]: Okay. Actually linting remove it

Comment (avaidyam) [reply src/service/ActivityService.ts:14]: That is odd and unfortunate as it should not be doing that. If the linter changed it then for now it's fine but let's adjust the `tsconfig.json` to be as **strict as possible** and adjust the eslint/prettier config in `package.json` to be strict as possible as well without deleting typing info.

Comment (Linoy339) [reply src/service/ActivityService.ts:14]: Okay. I have pushed the changes

**Pull request merged**